src/components/error-dialog/index.tsx

```diff
@@ -33,10 +33,15 @@ export class ErrorDialog extends React.Component<Properties> {
           <div {...cn('header')}>{header}</div>
         </div>
         <div {...cn('body')}>{body}</div>
-        <div {...cn('footer')}>
-          <Button onPress={onClose} variant='text'>
-            Dismiss
+        <div {...cn('footer', classNames({ 'is-single-button': !hasLink }))}>
+          <Button {...cn('button')} onPress={onClose} variant={hasLink ? 'text' : 'primary'}>
+            Back to my conversations
```

Comment:
Yeah this would need to be updated to wrap up the generalisation of the error dialog component. I left this for now as we're only using the error dialog for accessing a conversation. In the designs the copy doesn't change. Shall note to update at some point this sprint, if not immediate